src/libmongoc/src/mongoc/mongoc-client-side-encryption.c

```diff
@@ -888,6 +888,18 @@ _do_spawn (const char *path, char **args, bson_error_t *error)
 }
 #else
 
+/**
+ * @brief Exit the calling process *immediately* without running any per-process
+ * cleanup handlers.
+ *
+ * @param status The exist status for the process.
+ */
+static void
+fast_exit (int status)
+{
+   _exit (status);
```

Comment:
Why `_exit ()` (POSIX only) and not `_Exit ()` ([C99 equivalent](https://man7.org/linux/man-pages/man2/exit.2.html))?

Reply:
(See other comment)